tool/cli.js

```diff
@@ -83,6 +83,21 @@ program
     })
   )
 
+  .command("fix-redirects", "Consolidate/fix redirects")
+  .argument("<locale...>", "Locale", {
+    default: DEFAULT_LOCALE,
```

Comment:
```suggestion
    default: [DEFAULT_LOCALE],
```
otherwise, the default (in `const {locale } = args;`) makes `locale` become just a string (`en-US`).